Gopkg.toml

```diff
@@ -21,7 +21,8 @@ required = [
 
 [[override]]
   name = "knative.dev/pkg"
-  branch = "master"
+  source = "github.com/nachocano/pkg"
```

Comment:
/hold 
until Nacho can finish upstreaming that change into pkg.

Reply:
/hold cancel
changed has been upstreamed